jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -683,8 +685,11 @@ function configure_server_state() {
     local kieServerRepo="${HOME}/.kie/repository"
     JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.server.repo=${kieServerRepo}"
 
+
+#         unset
     # see above: configure_server_env / kieserver-env.sh / setKieEnv
-    if [ "${KIE_SERVER_CONTAINER_DEPLOYMENT}" != "" ]; then
+    if [ "x${KIE_SERVER_CONTAINER_DEPLOYMENT}" != "x" ]; then
+
         # ensure all KIE dependencies are pulled for offline use (this duplicates s2i process; TODO: short-circuit if possible?)
         $JBOSS_HOME/bin/launch/kieserver-pull.sh
```

Comment:
Isn't there missing hand over of env var `KIE_SERVER_DISABLE_KC_PULL_DEPS ` to the script?

Reply:
no, it will be handled on the pull and verify scripts.